Trans es templating Issue #22

oyepez003 opened the pull request:
Templating 100% traducido. Issue #22

oyepez003 committed:
Merge pull request #1 from oyepez003/trans_es_code_style_guide

Trans es code style guide

oyepez003 committed:
08-01-01-Templating.md 100% traducido. Issue #22

08-01-01-Templating.md 100% traducido. 

Issue #22

oyepez003 committed:
08-02-01-Benefits.md 100% traducido. Issue #22

08-02-01-Benefits.md 100% traducido. 

Issue #22

oyepez003 committed:
08-03-01-Plain-PHP-Templates.md 100% traducido.

08-03-01-Plain-PHP-Templates.md 100% traducido.

Issue #22

oyepez003 committed:
08-04-01-Compiled-Templates.md 100% traducido.

08-04-01-Compiled-Templates.md 100% traducido.

Issue #22

oyepez003 committed:
08-05-01-Further-Reading.md 100% traducido.

08-05-01-Further-Reading.md 100% traducido.

Issue #22

oyepez003 commented:
@joseayram @3rn3st0 A la espera de sus comentarios y correcciones si aplican.

3rn3st0 commented:
En el archivo **08-03-01-Plain-PHP-Templates.md**, donde dice: `"...en realidad es uno lenguaje propio para plantillas..."`, debería decir `"...en realidad es un lenguaje propio para plantillas..."`.

En el mismo párrafo, luego del punto y seguido, hay un _"tuteo"_. Hasta donde he podido observar, al lector se le trata de Usted en toda la documentación, no deberíamos hacer cambios al respecto o, en todo caso, la idea sería definir si tuteamos al usuario o si lo tratamos de Usted.

En el archivo **08-04-01-Compiled-Templates.md**, sugiero que el el texto donde dice:
- `"...ofreciendo una nueva sintaxis..."`, se cambie por `"...ofreciendo nuevas sintaxis..."`.
- `"...para ser más fácil de escribir..."`, se cambie por `"...para ser más fáciles de escribir..."`.

NOTA: Luego vuelvo a revisar, no sea que se me haya pasado por alto algún detalle.

oyepez003 commented:
@3rn3st0 

Esto creo que es importante definirlo "la idea sería definir si tuteamos al usuario o si lo tratamos de Usted."

Yo creo para lo que se esta traduciendo no deberia ser tan formal como nombrar "Usted" pero tampoco tan informal como para decir "Tu".... Creo que se puede obviar la palabra "Tu" y "Usted" 
Por ejemplo:

1) Usted puede realizar tal acción.
2) Tu puedes realizar tal acción.
3) Puedes realizar tal acción.

Creo que la mejor opcion seria la 3. Porque no estamos traduciendo un libro sino ideas y recomendaciones de un desarrollador para otro desarrollador.

No se que opinan.

@joseayram @Ashrey @aasanchez

oyepez003 committed:
Cambios recomendados por @3rn3st0

oyepez003 committed:
Cambios recomendados por @3rn3st0

oyepez003 commented:
@3rn3st0 @joseayram Listo los cambios, queda pendiente lo de "definir si tuteamos al usuario o si lo tratamos de Usted".

3rn3st0 commented:
@oyepez003 me parece ideal lo de la tercera opción en cuanto al tuteo o el trato de usted. En lo personal hice las traducciones con "usted" siguiendo lo que ya venía de la primera versión de PtRW. Por mi está bien si tuteamos obviando el tú, cuando no sea relevante.

oyepez003 commented:
@3rn3st0 @joseayram @aasanchez @Ashrey Hola chicos recuerden este PR.

joseayram commented:
@oyepez003 Me gusta bastante la traducción y los extras que le colocas en algunos casos me parecen fundamentales, por ejemplo: MVC (por sus siglas en inglés)... y así otros más que alcancé a leer.

Acá la única observación la tengo en _posts/08-02-01-Benefits.md

"son divididos en piezas más pequeños y reutilizables comúnmente llamadas" por "son divididos en piezas más **pequeñas** y **reutilizables,** comúnmente llamadas..."

oyepez003 committed:
[update] Observaciones.

Se cambia: "son divididos en piezas más pequeños y reutilizables comúnmente llamadas" por "son divididos en piezas más pequeñas y reutilizables, comúnmente llamadas..."

oyepez003 commented:
@joseayram Listo la última observación.
